fdbserver/DDShardTracker.actor.cpp

```diff
@@ -972,7 +972,7 @@ static bool shardBackwardMergeFeasible(DataDistributionTracker* self, KeyRange c
 		return false;
 	}
 
-	if (self->bulkLoadEnabled && self->bulkLoadTaskCollection->overlappingTask(prevRange)) {
+	if (self->bulkLoadEnabled && self->bulkLoadTaskCollection->onBulkLoading(prevRange)) {
```

Comment:
What does onBulkLoading mean? Does it mean we are in the middle of a bulk loading? If so, just say ->bulkLoading(prevRange)